src/components/Reports/Reports.js

```diff
@@ -17,17 +17,28 @@ import { addDays } from '../../utils';
 
 const periodSelectedDaysDefault = 7;
 
+const getDateNow = (intl) => {
+  return new Date(
+    intl.formatDate(Date.now(), {
```

Comment:
We have talked about removing _intl_ dependency in this function.

Also about renaming the function to `getCurrentDate` or convert this to something like `extractPureDate(date)` or `getBeginingOfDay(date)`.